docs/API.md

```diff
@@ -74,6 +74,41 @@ Captures, also known as capturing groups, extract and store parts of the matched
 > [!NOTE]
 > TS Regex Builder does not have a construct for non-capturing groups. Such groups are implicitly added when required. E.g., `zeroOrMore(["abc"])` is encoded as `(?:abc)+`.
 
+### `backreference()`
```

Comment:
Using ordinal backreferences accurately might be problematic in case of more complex expressions, nesting, etc. Therefore, I think we can drop them without loosing any functionality to the user trying to build maintainable regexes.